util/csvresponse.py

```diff
@@ -57,12 +64,10 @@ def _create_filter_list(self,keys):
         :param keys: list of keys from the dataset.
         :return: set containing keys to filter from the dataset.
         """
-        # default parameters -- move to class variable?
-        default = ['time', 'deployment', 'lat', 'lon']
-
         # initialize param list to include requested and default parameters
         missing_params = []
-        params_to_include = []
+        params_to_include = DEFAULT_PARAMETERS
```

Comment:
not sure about python, but i think assignment is by reference.  if params_to_include are modified it could also modify default_parameters, if DEFAULT_PARAMETERS are put someplace common it may make sense to make sure and grab a copy of the values instead of the actual values.

Reply:
You are correct. Will change to use the list constructor method.